src/store/ConfigStore.ts

```diff
@@ -16,8 +16,11 @@ import { INITIAL_STATUS_STATE } from '../reducers/HelpDropdownState';
 
 declare const window;
 
+const webRoot = (window as any).WEB_ROOT ? (window as any).WEB_ROOT : undefined;
+const persistKey = webRoot && webRoot !== '/' ? webRoot.substring(1) : 'root';
```

Comment:
This is a great idea! could we also maybe change 'root' to 'kiai-root'

Reply:
> 
> Would it make sense here to do a composite key, that starts with e.g. 'io.kiali' to indicate that whatever you use it points to Kiali?

That is great idea to using composite key to ensure it is unique. I would like to using 'kiali' as a prefix. Thanks.